test/tracerEq/test_bcs_2d.py

```diff
@@ -116,9 +118,12 @@ def run(refinement, **model_options):
     options.solve_tracer = True
     options.tracer_only = True
     options.horizontal_diffusivity = nu
-    options.use_limiter_for_tracers = True
+    options.horizontal_diffusivity_scale = nu
+    options.horizontal_velocity_scale = Constant(0.0)
```

Comment:
Just curious: is there a reason to set this to 0?

Reply:
Well just because I figured that was a characteristic velocity for that test case, which is just a diffusion problem. Setting it to zero will switch SUPG off.